R/zchunk_L105.nh3_an_USA_S_T_Y.R

```diff
@@ -53,19 +55,63 @@ module_emissions_L105.nh3_an_USA_S_T_Y_DISABLED <- function(command, ...) {
     # For more information, see https://github.com/JGCRI/gcamdata/wiki/Name-That-Function
     # ===================================================
 
+    # Process EPA NH3 emissions estimates:
+    # Map to GCAM sector and Technology, convert to Tg and aggregate
+    # Notes:
+    # - when mapping from EPA sectors to GCAM sectors and technologies there should be NAs
+    #   (don't map very aggregate or very detailed EPA sectors to avoid double counting)
+
+    L105.nh3_tg_USA_an_Sepa_F_Yh <- EPA_NH3 %>%
+      left_join(EPA_tech, by = c("Source_Category" = "EPA_Category")) %>%
+      filter(sector == 'Animals',!is.na(fuel)) %>%
+      select(-Source_Category_Raw, -Source_Category) %>%
+      gather(year, value, -sector, -fuel) %>%
+      mutate(year = as.numeric(year)) %>%
+      filter(year %in% NH3_HISTORICAL_YEARS) %>%
+      mutate(value = value * CONV_TST_TG) %>%  # convert from thousand short tons to Tg
+      dplyr::rename(emissions = value)
+
+    # Process FAO production data:
+    # Subset US data, aggregate to EPA sectors (all animal production)
+
+    L105.out_Mt_USA_an_C_Sys_Fd_Yh <- L107.an_Prod_Mt_R_C_Sys_Fd_Y %>%
+      filter(GCAM_region_ID == 1) %>%
```

Comment:
There's a global constant defined for this--use it please. `GCAM_REGION_USA` or something like that.